azure/container.go

```diff
@@ -32,6 +34,11 @@ func (c *container) Name() string {
 	return c.id
 }
 
+func (c *container) PreSignRequest(_ context.Context, _ stow.ClientMethod, _ string,
+	_ stow.PresignRequestParams) (url string, err error) {
+	return "", fmt.Errorf("unsupported")
```

Comment:
if azure won't work, does this mean we're going to have to have logic in flytectl to detect if we're trying to use azure storage, and then switch back to using the current logic?

Reply:
Implemented